{{ cookiecutter.package_name|slugify }}/pyproject.toml

```
@@ -58,10 +58,11 @@ uvicorn = { extras = ["standard"], version = "^0.16.0" }
 # https://python-poetry.org/docs/master/managing-dependencies/
 # TODO: Split in `tool.poetry.group.dev` and `tool.poetry.group.test` when Poetry 1.2.0 is released.
 [tool.poetry.dev-dependencies]
-cruft = "^2.10.1"
+bandit = "1.7.2"  # TODO: remove this line once this is solved: https://github.com/PyCQA/bandit/issues/841
```

- I think this one is the original bandit issue (the one you link to appears to be a duplicate): https://github.com/PyCQA/bandit/issues/837. Doesn't look like bandit will solve this though since they don't consider the thing that is broken as part of their public API.
- And this is the flake8-bandit issue: https://github.com/tylerwince/flake8-bandit/issues/21. Would refer to this one instead, hoping that they can fix it there.
- Would also relax the bandit constraint a bit just in case an exact pin could cause the environment to be unsolvable:
```suggestion
bandit = "<1.7.3"  # TODO: Remove after https://github.com/tylerwince/flake8-bandit/issues/21
```

---

I updated the referenced issue.
Regarding the pinning, I can't set it to `"<1.7.3"` for now because both `1.7.3` and the latest version `1.7.4` still have the bug. But the hard pinning should not be an issue on the long run since the todo suggests to remove that dependency and trust flake8-bandit to install the right version in the future.

---

Resolving this conversation, and opened a new one related to it.